src/alexapi/device_platforms/chip.py

```diff
@@ -0,0 +1,118 @@
+import time 
+import os
+
+import CHIP_IO.GPIO as GPIO
+# NOTE: at this point CHIP_IO.GPIO needs to be built following "manual install" instructions at
+# https://github.com/xtacocorex/CHIP_IO
+# Those instructions are also included here:
+#   sudo apt-get update
+#   sudo apt-get install git build-essential python-dev python-pip flex bison -y
+#   git clone https://github.com/atenart/dtc
+#   cd dtc
+#   make
+#   sudo  make install PREFIX=/usr
+#   cd ..
+#   git clone git://github.com/xtacocorex/CHIP_IO.git
+#   cd CHIP_IO
+#   sudo python setup.py install
+#   cd ..
+#   sudo rm -rf CHIP_IO
+
+
+from baseplatform import BasePlatform
+import alexapi.bcolors as bcolors
+
+
+class ChipPlatform(BasePlatform):
+
+	def __init__(self, config):
+		super(ChipPlatform, self).__init__(config)
+
+		self.__config = config
+		self.__pconfig = config['platforms']['chip']
```

Comment:
Thanks.  Yeah, I still had some stuff to do.

I think I've captured it all though--added notes to the wiki too.  See what you think now.